kitsune/lib/babel.py

```diff
@@ -30,7 +12,7 @@ def generate_option(value):
     everything is based on strings.
     """
     if isinstance(value, bool):
-        return f"{bool(value)}".lower()
+        return str(value).lower()
```

Comment:
Doesn't this expect a bool as a return value?

Reply:
No, it expects strings, which seems a bit strange to me as well (see https://github.com/pallets/jinja/blob/e740cc65d5c54fbebb0f3483add794ec2b47187f/src/jinja2/ext.py#L813).